ex03.ex

```diff
@@ -92,17 +96,25 @@ defmodule Ex03 do
   a function that checks for the equality of two lists. You don't
   need to consider nested lists.
 
-      iex> Ex03.list_equal [ 1, 2, 3 ], [1, 2, 3]
+      iex> Ex03.list_equal([ 1, 2, 3 ], [1, 2, 3])
       true
-      iex> Ex03.list_equal [ 1, 2, 3 ], [1, 2, 3, 4]
+      iex> Ex03.list_equal([ 1, 2, 3 ], [1, 2, 3, 4])
       false
-      iex> Ex03.list_equal [ 1, 2, 3 ], [3, 2, 1]
+      iex> Ex03.list_equal([ 1, 2, 3 ], [3, 2, 1])
       false
 
+ cond do
+           h + Enum.count(t) == h2 + Enum.count(t2) -> true
+           h == h2 -> list_equal(t,t2)
+       end
   """
 
-  def list_equal . . . "your code"
-
+  def list_equal([], []), do: true
```

Comment:
# -3

This is a little messy.

Again, there's no need for the equality test on the two heads. You could just write:

~~~ elixir
def list_equal([], []), do: true
def list_equal([h | t1], [h | t2]), do: list_equal(t1, t2)
def list_equal(_, _), do: false
~~~

There's no need for the count test.